app/controllers/application.js

```diff
@@ -11,7 +11,8 @@ export default Ember.Controller.extend({
   },
 
   handleStorageDisconnect: function() {
-    console.log('rs disconnected, transition to welcome');
+    localStorage.removeItem('webmarks:tags');
```

Comment:
You might want to move this into the `storage` service as well, similar to all other `localStorage` operations.